src/System.Diagnostics.EventLog/src/System.Diagnostics.EventLog.csproj

```diff
@@ -8,7 +8,6 @@
     <Configurations>net461-Debug;net461-Release;netcoreapp-Windows_NT-Debug;netcoreapp-Windows_NT-Release;netcoreapp2.0-Windows_NT-Debug;netcoreapp2.0-Windows_NT-Release;netfx-Debug;netfx-Release;netstandard-Debug;netstandard-Release</Configurations>
   </PropertyGroup>
   <ItemGroup Condition="'$(TargetsNetCoreApp)' == 'true'">
-    <Compile Include="System\Diagnostics\CompModSwitches.cs" />
```

Comment:
Diff for removing System.Diagnostics.TraceSource missing in this csproj

Reply:
It's still required.  This assembly defines an EventLogTraceListener that derives from TraceListener that's in System.Diagnostics.TraceSource.dll.